tests/transactional/enable_fips.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: FIPS
+# Summary: Enable FIPS on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use Mojo::Base "consoletest";
+use testapi;
+use transactional;
+use bootloader_setup qw(change_grub_config);
+use version_utils qw(is_sle_micro);
+
+sub run {
+    select_console 'root-console';
+
+    # make sure fips is not enabled
+    assert_script_run("grep '^0\$' /proc/sys/crypto/fips_enabled");
+
+    # install fips pattern
+    record_info('INFO', "Installing FIPS pattern");
+    if (is_sle_micro) {
+        trup_call("pkg install -t pattern microos-fips");
+    } else {
+        trup_call("pkg install -t pattern fips");
+        trup_call("--continue pkg install fipscheck");
+    }
+    check_reboot_changes;
```

Comment:
Is it intentional to reboot twice? If so, can you add a comment why?

Reply:
No, just following the docs. But it can be done in 1 snapshot.